jmx_prometheus_httpserver_common/src/main/java/io/prometheus/jmx/WebServer.java

```diff
@@ -29,5 +35,12 @@ public static void main(String[] args) throws Exception {
      new BuildInfoCollector().register();
      new JmxCollector(new File(args[1]), JmxCollector.Mode.STANDALONE).register();
      new HTTPServer(socket, CollectorRegistry.defaultRegistry);
+
+     System.out.println(
+       String.format("%s | %s | INFO | %s | %s",
+         SIMPLE_DATE_FORMAT.format(new Date()),
+         Thread.currentThread().getName(),
+         WebServer.class.getName(),
+         "Running"));
```

Comment:
Adding a message to the `WebServer` is not as critical as adding a message to the agent. However, if you remove it from the agent you should remove it here too for consistency.

Reply:
Made the change to the Java agent

Reply:
Made the change to the Java agent